Tools/PlatformHaiku.cmake

```diff
@@ -1,5 +1,8 @@
 add_subdirectory(TestRunnerShared)
-add_subdirectory(DumpRenderTree)
-add_subdirectory(HaikuLauncher)
 add_subdirectory(ImageDiff)
```

Comment:
ImageDiff can be removed from here (already done in the main CMakeLists.txt for the tools)